api/tests.edn

```diff
@@ -1,5 +1,6 @@
 #kaocha/v1
     {:tests [{:id           :integration
+              :skip-meta    [:pending]
```

Comment:
This is quite a nice feature. Can we use something like `exclude` or `filter` - `pending` doesn't quite capture the general case i think!

Reply:
Thanks! I think `exclude` is a better name, will update.